_sass/_base.scss

```diff
@@ -175,18 +175,18 @@ header {
     width: 100%;
     min-height: auto;
     @include background-cover;
-    background: center no-repeat url("../img/header-960.jpg");
+    background: center 60px no-repeat url("../img/header/header-960.jpg");
     @media (min-width: 961px) {
-        background: center no-repeat url("../img/header-1920.jpg");
+        background: center -60px no-repeat url("../img/header/header-1920.jpg");
     }
     @media (min-width: 1921px) {
-        background: center no-repeat url("../img/header-3840.jpg");
+        background: center -60px no-repeat url("../img/header/header-3840.jpg");
```

Comment:
For some reason `header-3840.jpg` is displayed somehow zoomed-in and hence looks weird.
![Screenshot_20200503_000927](https://user-images.githubusercontent.com/2820734/80893434-7f995f00-8cd2-11ea-9514-9c3005b1c427.png)

Reply:
Ah so einen großen Bildschirm habe ich nicht.